### DISCUSSION TITLE
Not getting NFTs

### DISCUSSION THREAD

**Discussion by ShubhSensei** [Cyfrin/foundry-full-course-cu#120] [Q&A]
I'm not able to collect NFTs, I'm getting this error after clicking the " write " button under renounceOwnership function.

How to solve challenge 1? I think the problem lies there.

<img width="902" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/d8be0833-1b9c-4b0d-be70-7572dd127b38">

**Comment by npa528** [comment]
@SensationalShubham You need to use solveChallenge method don't click renounceOwnership (as seen in your screenshot), it will not work obviously.

**Reply by ShubhSensei** [reply]
I already used it & It's not working. Now what?

**Reply by alymurtazamemon** [reply]
@SensationalShubham Did you mint it before? Because you cannot mint twice

And your metamask show you are calling the `renounceOwnership` function, instead you have to call `solveChallenge`

**Reply by ShubhSensei** [reply]
Look, it's not working instead. And I didn't Mint it before.
(Btw what I did was just scan the QR code and click on "write" button just after that.  Do I need to write/Read something or do something else to work out )

<img width="899" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/d714f9be-7d5e-40cb-afae-9b76b3ab268c">

**Reply by alymurtazamemon** [reply]
@SensationalShubham If it is lesson 1 then you just need to call it with or without twitter handle.

**Reply by ShubhSensei** [reply]
Yes, I'm only just calling 'write" button of solveChallenge & still this error pops up. The above image is the latest one.

**Reply by alymurtazamemon** [reply]
@SensationalShubham here you can see it is working for me

Is the same address `0x25056312685339b49e1d1C5a0b72Ff9eff13AF77`? It is on sepolia.

<img width="1463" alt="Screenshot 2023-06-20 at 10 31 57 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/56123405/f7af1e54-4d24-40af-ba46-f96db2214b20">

**Reply by ShubhSensei** [reply]
It's giving me this error 

<img width="869" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/91625601/64192cfd-46fd-48e1-9a0d-9d63c60c4431">

**Reply by cromewar** [reply]
Seems your metamask is not interacting properly with the contract, have you being able to solve this?
This should work just fine using your twitter handle:
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/20199180/1b0c06b1-71e9-4d7f-b545-d7221a61b772)
